public/app/plugins/panel/bargauge/module.tsx

```diff
@@ -52,7 +52,7 @@ export const plugin = new PanelPlugin<Options>(BarGaugePanel)
           ],
         },
         defaultValue: defaultOptions.namePlacement,
-        showIf: (options) => options.orientation !== VizOrientation.Vertical,
+        showIf: (options) => options.orientation === VizOrientation.Horizontal,
```

Comment:
I think we should keep the previous behavior here:

The name placement is applicable for both the Auto + Horizontal orientations (as horizontal orientation can occur in the "auto" layout mode)


https://github.com/grafana/grafana/assets/22381771/d1029201-52dd-4acb-86fb-cd56a93b06d9